lib/sift/client.rb

```diff
@@ -1,17 +1,22 @@
 require 'httparty'
 require 'multi_json'
 
+require_relative "./client/decision"
```

Comment:
require_relative was added in 1.9.2 — are we trying to maintain 1.8.7 compatibility?